src/PKSim.Core/Model/ExpressionProfile.cs

```diff
@@ -1,13 +1,40 @@
-﻿using OSPSuite.Core.Domain;
+﻿using System.Linq;
+using OSPSuite.Core.Domain;
 
 namespace PKSim.Core.Model
 {
    public class ExpressionProfile : PKSimBuildingBlock
    {
+      public const string DUMMY_MOLECULE_NAME = "%DUMMY%";
+
+      private string _category;
       public virtual Species Species { get; set; }
 
       public ExpressionProfile() : base(PKSimBuildingBlockType.ExpressionProfile)
       {
       }
+
+      public virtual string MoleculeName => Molecule?.Name;
+
+      public virtual string Category
+      {
+         get => _category;
+         set
+         {
+            _category = value;
+            RefreshName();
+         }
+      }
+
+      public Individual Individual { get; set; }
```

Comment:
@Yuri05 I am saving a full Individual for each profile. However the individual has no parameters (only container structure)